apps/src/applab/Visualization.jsx

```diff
@@ -0,0 +1,32 @@
+var applabConstants = require('./constants');
+
+var styles = {
+  hidden: {
+    display: 'none'
+  }
+};
```

Comment:
I considered that. I'm not sure what else would go in there though, and it seems a little silly to have a separate file just for this.